src/components/taskDetails/index.tsx

```diff
@@ -331,14 +320,38 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
                                     )}
                                 />
                             </TaskContainer>
-                            <TaskContainer
-                                hasImg={false}
-                                title="Request for task"
-                            >
-                                <button onClick={taskRequestHandle}>
-                                    Request for task
-                                </button>
-                            </TaskContainer>
+                            {isDevModeEnabled && (
```

Comment:
`isDevMode` is fine instead of `isDevModeEnabled`, if `isDevMode` is true then it is pointing to dev mode is enabled!